todo/views/team.py

```
@@ -0,0 +1,85 @@
+from rest_framework.views import APIView
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)